src/screens/Donate/Donate.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Fragment } from 'react';
+import { Grid } from '@material-ui/core';
+import { Icon } from '@iconify/react';
+import testTube from '@iconify/icons-twemoji/test-tube';
+import loveLetter from '@iconify/icons-twemoji/love-letter';
+import deliveryTruck from '@iconify/icons-twemoji/delivery-truck';
+import {
+  EmbedWrapper,
+  ContentWrapper,
+  SectionHeader,
+  ContentSection,
+  ListHeader,
+  BodyCopy,
+  ListSection,
+} from 'screens/Donate/Donate.style';
+import PartnersSection from 'screens/HomePage/PartnersSection';
+
+const roadmapListContent = [
+  {
+    header: 'Improve our data quality',
+    listItems: [
+      'More accurate data',
+      'More transparency of data sources',
+      'Encourage states to adopt better contact tracing data practices',
+    ],
+  },
+  {
+    header: 'Inspire more action',
+    listItems: [
+      'Localized recommendations for masking, schools, and businesses based on new daily cases and positivity rate',
+      'Localized masking compliance; illustrate impact of increasing compliance',
+      'Continually survey our audience to understand their biggest needs as they change',
+    ],
+  },
+  {
+    header: 'Expand our reach',
+    listItems: [
+      'More educational content so that more people can understand our Warning System',
+      'Closer relationships with high-risk and underserved communities',
+      'More tools for scientists and journalists to help uncover new trends and correlations',
+    ],
+  },
+];
+
+const Donate = () => {
+  return (
+    <Fragment>
+      <EmbedWrapper>
+        <iframe
+          title="Donation embed"
```

Comment:
Consider changing the title to something more meaningful `Donate to Covid Act Now on GiveButter`